gems/yard/CVE-2019-1020001.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: yard
+url: https://github.com/lsegal/yard/security/advisories/GHSA-xfhh-rx56-rxcr
+date: 2019-07-02
+title: Moderate severity vulnerability that affects yard
+description: "## Possible arbitrary path traversal and file access via `yard server`\r\n\r\n###
```

Comment:
Instead of \r\n, please use new lines and break this description up. See other advisories.